.github/workflows/R-CMD-check.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     name: ${{ matrix.config.os }} (${{ matrix.config.r }})
 
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
No need to run all tests if one of them fails. This stops all of them once there is a failure. These are free instances so we don't have to do this, but let's pretend this is saving the environment :)